feature/experimental/system/gnmi/benchmarking/ate_tests/full_configuration_replace_test/README.md

```diff
@@ -0,0 +1,25 @@
+# gNMI-1.2: Benchmarking: Full Configuration Replace 
+
+## Summary
+
+Measure performance of full configuration replace.
+
+## Procedure
+
+Configure DUT with:
+ - Maximum number of interfaces to be supported.
+ - Maximum number of BGP peers to be supported.
+ - Maximum number of IS-IS adjacencies to be supported.
```

Comment:
I find these statements about "maximum number of ..." slightly confusing. In the code you configure 1 BGP neighbor & 1 ISIS adjacency per ATE interface. Can we rephrase it (to avoid potential confusion with scalability testing / maximum number of supported interfaces/adjacencies on a node)?

Reply:
Updated the description; please see if it is ok.